docs/UserGuide.md

```diff
@@ -166,6 +166,12 @@ AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.
 If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
 </div>
 
+### Showing the schedule for a specific day : 'schedule'
+
+Shows the contacts of students who have classes on a specific day.
+
+Format: 'schedule DAY'
+
```

Comment:
Maybe you can add an example here like:
```
Examples:
* Schedule Monday
```
to match with the above format

Reply:
Other than that LGTM

Reply:
ok added